### PR TITLE
[chore] pre-commit autoupdate

### PR DESCRIPTION
**Pull request by Pirols**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -37,7 +41,6 @@ ignore = [
 
     "PLR2004",  # magic-value-comparison; not all comparisons to int or str are magic
     "TRY003",  # raise-vanilla-args; we can live without it
-    "TRY200",  # reraise-no-cause; not python2 compatible
```
Comment (KangOl) [inline pyproject.toml:40]: Why removing this?

**Comment by KangOl** [comment]
New ruff version expected soon. Wait until v0.3 is released.

**Comment by Pirols** [comment]
> New ruff version expected soon. Wait until v0.3 is released.

It's out. PR updated accordingly.

**Comment by KangOl** [comment]
upgradeci skip

**Comment by KangOl** [comment]
@robodoo rebase-ff r+

**Pull request closed without merging**